app/models/automate_workspace.rb

```diff
@@ -18,4 +18,33 @@ def merge_output!(hash)
     save!
     self
   end
+
+  def decrypt(object_name, attribute)
+    MiqAePassword.decrypt(encrypted_value(object_name, attribute))
+  end
+
+  def encrypt(object_name, attribute, value)
+    self.output ||= {}
+    self.output.store_path('objects', object_name, attribute, "password::#{MiqAePassword.encrypt(value)}")
+    save
```

Comment:
I feel like this shouldn't do the saving, but I'm not sure

Reply:
Also, what is the expected return value here?  Should this be doing save! instead, and if so, would it be better to just use merge_output!